x-pack/plugins/console_extensions/spec/overrides/xpack.info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_xpack": {
+    "url_params": {
+      "categories": [
+        "build",
+        "license",
+        "features"
+      ],
+      "human": [
```

Comment:
Is this supposed to show up as an option?

![image](https://user-images.githubusercontent.com/1238659/41483105-a6b607aa-708c-11e8-8b41-c2cd836b6752.png)